.env.example

```diff
@@ -1,23 +1,44 @@
-GOOGLE_API_KEY=
-GOOGLE_CLIENT_KEY=
+# Env vars
 
-OFSTED_API_KEY=
-OFSTED_FEED_API_ENDPOINT=
+# ------------------ deprecated
+
+#INSTANCE=
+#OUTPOST_INSTANCE_NAME=
+
+# ------------------ outpost
+
+FEEDBACK_FORM_URL=
+
+
+# ------------------ scout
+
+SCOUT_URL=
+#SCOUT_BUILD_HOOK= 
```

Comment:
Is this one deprecated? Why is it commented out?

Reply:
I have added a comment, its not required in development, but I also always commented it out because there was a bug in the code: PR here https://github.com/wearefuturegov/outpost/pull/242

Reply:
Cool, thanks!